src/ontology/mondo-edit.obo

```diff
@@ -443612,21 +443581,14 @@ replaced_by: MONDO:0014958
 
 [Term]
 id: MONDO:0044655
-name: c12orf65-related combined oxidative phosphorylation defect
-comment: Reason of obsoletion: out of scope - MONDO:excludeGroupingClass. This is part of a large mass-obsoletion effort for ORDO terms.
-subset: disease_grouping
-subset: obsoletion_candidate
-subset: ordo_group_of_disorders
+name: obsolete c12orf65-related combined oxidative phosphorylation defect
 synonym: "C12ORF65-related COXPD" EXACT [Orphanet:497623]
-xref: Orphanet:497623 {source="MONDO:equivalentTo"}
-is_a: MONDO:0016387 ! mitochondrial oxidative phosphorylation disorder
-intersection_of: MONDO:0000001 ! disease
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/26784 ! MTRFR
+xref: Orphanet:497623 {source="MONDO:obsoleteEquivalent"}
 relationship: excluded_subClassOf MONDO:0000508 {source="MONDO:Redundant", source="Orphanet:497623"} ! syndromic intellectual disability
-relationship: has_characteristic MONDO:0021136 {source="MONDO:0015162", source="MONDO:0015983"} ! rare
```

Comment:
the pipeline is doing this. There really isn't any harm to leaving the excluded subclass but it could be removed. @sabrinatoro @matentzn do you think we should modify the pipeline to remove the excluded_subClassOf  annotations from obsolete terms?

Reply:
What is confusing here is that both relationships look similar in obo format, but are entirely different in OWL - the one, the removed one - is a logical axiom. The other an annotation assertion. I think it should be removed in this case, but we can do this as part of a general spring clean.

Reply:
And making the question more broad, should only Object Properties (like `has_characteristic`) be removed in the pipeline?

Reply:
I think some annotation properties like xrefs, labels can be conserved. Possibly even synonyms, no harm done. But I think we should document which and keep them to a minimum.

Reply:
Looks like we were posting around the same time :)

Ok, so relationships using an Object property are `logical` and relationships using an Annotation property are not. Thanks!